bayes/vb.py

```diff
@@ -420,8 +437,8 @@ def run(self, model_error, param0, noise0=None, **kwargs):
                 r = 2 / (m[index_ARD] ** 2 + np.diag(L_inv)[index_ARD])
                 d = 0.5 * np.ones(n_ARD_param)
 
-            logger.debug(f"current mean: {m}")
-            logger.debug(f"current precision: {L}")
+            logger.debug(f"scaled current mean: {m}")
```

Comment:
I think for a physical understanding it might be helpful to also log the unscaled values.